content/v2.0/write-data/delete-data.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Delete data
+list_title: Delete data
+description: >
+  Delete data in the InfluxDB UI.
+menu:
+  v2_0:
+    name: Delete data
+    parent: Write data
+weight: 104
+v2.0/tags: [delete]
+related:
+  - /v2.0/reference/syntax/delete-predicate/
+  - /v2.0/reference/cli/influx/delete/
+---
+
+## Delete data in the InfluxDB UI
+
+You can only delete data from buckets you've created. You cannot delete data from system buckets.
+
+### Delete data from buckets
+
+1. Click **Load Data** in the navigation bar.
+
+    {{< nav-icon "load data" >}}
+
+2. Select **Buckets**.
+3. Next to the bucket with data you want to delete, click **Delete Data by Filter**.
+4. In the **Delete Data** window that appears:
+  - Select a **Target Bucket** to delete data from.
+  - Enter a **Time Range** to search.
+  - Click **+ Add Filter** to filter by tag key and value pair.
+  - Operator other than equals?
```

Comment:
- Select operator other than equals (remove question mark?)

Reply:
Whoops, was supposed to delete that!